dll/win32/syssetup/lang/de-DE.rc

```diff
@@ -179,12 +179,12 @@ BEGIN
     IDS_COMPUTERSUBTITLE "Sie müssen einen Namen und ein Adminstratorpasswort für Ihren Computer festlegen."
     IDS_LOCALETITLE "Regionale Einstellungen"
     IDS_LOCALESUBTITLE "Sie können ReactOS auf unterschiedliche Regionen und Sprachen einstellen."
-    IDS_LOCALETEXT "The Standards and formats are set to %s, and the location is set to %s."
-    IDS_LAYOUTTEXT "Your default text input language and method is: %s keyboard layout"
+    IDS_LOCALETEXT "Die Einstellung für Standards und Formate ist %s, und der Standort ist %s."
```

Comment:
Sind

Reply:
Nevermind. I just looked at the Screenshot and it is OK as is.

Reply:
1.) I oriented strictly on XPSP3 here,
2.) "ist" refers to the subject "die Einstellung" of this sentence (and not to "Standards und Formate"). Therefore "ist" is indeed the correct term. Deutsch-Leistungskurs ;-)